src/internet_identity/internet_identity.did

```diff
@@ -293,6 +293,8 @@ type IdentityAnchorInfo = record {
     devices : vec DeviceWithUsage;
     // Device registration status used when adding devices, see DeviceRegistrationInfo
     device_registration: opt DeviceRegistrationInfo;
+    // OpenID accounts linked to this anchor
+    openid_credentials: opt vec OpenIdCredential;
```

Comment:
This is used in the old API. Could you add it in the v2 API as well?

I believe in `IdentityInfo`.